Thoms' changes to licence section (#66)
DanNixon/sine2020_software_guidelines_paper: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex

An open source license is advisable from both a user and development perspective, details of available licenses are available online \cite{OSI_Licenses}.
The users will be able to download and use the software free of charge, removing a potential barrier in the uptake of the software.
The advantage of an open source license is that it allows anyone to contribute to the code base (but must not bypass any review process) and radically improves the available pool of developers.

> Furthermore, in research/academic communities open source software is becoming increasingly compulsory in order to prevent the loss of valuable source code \cite{foss-nature}.

Reviewer: What assertion does the reference support here?

Reviewer 2: none as far as I can see. We need a different reference here if any. The reference is useful elsewhere

Reviewer: > We need a different reference here if any.

I'm not convinced this sentence says anything that is so far outside of common sense to require external support.

Reviewer: ```suggestion
Furthermore, in research/academic communities open source software is becoming increasingly compulsory in order to prevent the loss of valuable source code.
```

> Like FAIR data, open source software is one of the keystones in the European Commission's push for open science \cite{EOSC}. 

Author: ```suggestion
The European Commission's push for open science \cite{EOSC} includes both FAIR data and open source software.  
```

Reviewer 2: ok (it would probably be wise of me to pinpoint where that is mentioned exactly)
